activestorage/test/models/variant_with_record_test.rb

```diff
@@ -105,9 +105,8 @@ class ActiveStorage::VariantWithRecordTest < ActiveSupport::TestCase
     users.reset
 
     assert_no_difference -> { ActiveStorage::VariantRecord.count } do
-      assert_queries(6) do
-        # 6 queries:
-        # users x 1
+      assert_queries(5) do
```

Comment:
Strange 🤔 Any idea why this changed?

Reply:
The `assert_queries` method originally counted for cached queries, now it is not. 
We can probably also have an option to configure if we count for cached queries or not, but not sure if it will be used much.